src/nvim/ex_docmd.c

```diff
@@ -6816,7 +6840,7 @@ void post_chdir(int local)
 
 
 /*
- * ":cd", ":lcd", ":chdir" and ":lchdir".
+ * ":cd", ":tcd", ":lcd", ":chdir", ":tchdir" and ":lchdir".
```

Comment:
@HiPhish It should complain *because you changed the line adjacent to the comment start*. It does not care about *original* comments, error suppresion works only as long as some line and its adjacent lines are not change.